denizen_scripts/HeroCraft/mobs/prefixes/breaking.dsc

```diff
@@ -0,0 +1,7 @@
+custom_mob_prefix_breaking:
+  Type: world
+  debug: false
+  events:
+    after entity_flagged:breaking damages player:
```

Comment:
probably needs a check to not disable air

Reply:
👍